dns/ddclient/src/opnsense/mvc/app/models/OPNsense/DynDNS/DynDNS.xml

```diff
@@ -126,6 +126,12 @@
                         <if>Interface</if>
                     </OptionValues>
                 </checkip>
+                <checkip_timeout type="IntegerField">
+                    <default>10</default>
+                    <Required>Y</Required>
+                    <MinimumValue>10</MinimumValue>
+                    <MaximumValue>60</MaximumValue>
+                </checkip_timeout>
```

Comment:
best update the model version so the system knows it needs an (automatic) migration